projects/standard-rulesets/src/breaking-changes/__tests__/range-changes.test.ts

```diff
@@ -0,0 +1,173 @@
+import { OpenAPIV3 } from '@useoptic/openapi-utilities';
+import { TestHelpers } from '@useoptic/rulesets-base';
+import { BreakingChangesRuleset } from '../index';
+
+describe('range changes', () => {
+  test('numeric range expanded', () => {
```

Comment:
It feels like we're missing test coverage here with different variants (increase / decrease, numeric / array, request / response). We can factor our a lot of the common test logic using `test.each` to handle a lot of the different variants of test. Example of this https://github.com/opticdev/optic/blob/main/projects/standard-rulesets/src/breaking-changes/__tests__/parameter-enum-breaking-changes.test.ts#L6-L8